### PR TITLE
Split navbar component

### PR DESCRIPTION
**Pull request by michael-albrecht-1**
- split navbar into 2 components navbar and header
- each option is displayed with an delay of 30ms when we click on "Profile" button in the header

### CODE REVIEW

**Review by SimonBoeuf — COMMENTED** [review]
I fixed some stuff in the `yarn.lock` file, it seems like you added dependencies to the `package.json` file, then removed them, but kept the updated `yarn.lock`

**Comment by SimonBoeuf** [comment]
Also, your build is failing because of an unresolved lint issue, we should always fix build issues before merging

image

File: src/components/UserLinks/UserLinks.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  const [areProfileActionsDisplayed, setAreProfileActionsDisplayed] = useState(
+    false
+  );
+  const handleCloseProfileLinks = () => {
+    setAreProfileActionsDisplayed(false);
+    setDisplayedProfileActionsList(initialDisplayedProfileActionsList);
+  }
+
+  const [displayedProfileActionsList, setDisplayedProfileActionsList] = useState(initialDisplayedProfileActionsList);
```
Comment (SimonBoeuf) [inline src/components/UserLinks/UserLinks.js]: Sorry but I don't understand what it does, can you explain ? I tried to remove it completely and it seems to work fine

``` javascript
const UserLinks = () => {

  const [areProfileActionsDisplayed, setAreProfileActionsDisplayed] = useState(
    false
  );
  const handleCloseProfileLinks = () => {
    setAreProfileActionsDisplayed(false);
  }

  return (
      <ul className="profil">
        {!areProfileActionsDisplayed ? (
          <li onClick={() => setAreProfileActionsDisplayed(true)}>
            <i className="far fa-user"></i>Profil
          </li>
        ) :
        <>
          <li>
            <Link to="/profile">
              <i className="fas fa-cog" />Gérer mon profil
            </Link>
          </li>
          <li onClick={() => firebase.auth().signOut()}>
            <i className="fas fa-sign-out-alt" />Se déconnecter
          </li>
          <li onClick={handleCloseProfileLinks}>
            <i className="fas fa-times" />Fermer
          </li>
        </>
        }
    </ul>
  );
};
```

**Comment by michael-albrecht-1** [comment]
> Also, your build is failing because of an unresolved lint issue, we should always fix build issues before merging
> 
> image

My linter didnt displayed any warning.

File: src/components/UserLinks/UserLinks.js (hunk 1 above)
Comment (michael-albrecht-1) [reply src/components/UserLinks/UserLinks.js]: > Sorry but I don't understand what it does, can you explain ? I tried to remove it completely and it seems to work fine
> 
> ```js
> const UserLinks = () => {
> 
>   const [areProfileActionsDisplayed, setAreProfileActionsDisplayed] = useState(
>     false
>   );
>   const handleCloseProfileLinks = () => {
>     setAreProfileActionsDisplayed(false);
>   }
> 
>   return (
>       <ul className="profil">
>         {!areProfileActionsDisplayed ? (
>           <li onClick={() => setAreProfileActionsDisplayed(true)}>
>             <i className="far fa-user"></i>Profil
>           </li>
>         ) :
>         <>
>           <li>
>             <Link to="/profile">
>               <i className="fas fa-cog" />Gérer mon profil
>             </Link>
>           </li>
>           <li onClick={() => firebase.auth().signOut()}>
>             <i className="fas fa-sign-out-alt" />Se déconnecter
>           </li>
>           <li onClick={handleCloseProfileLinks}>
>             <i className="fas fa-times" />Fermer
>           </li>
>         </>
>         }
>     </ul>
>   );
> };
> ```

2021-03-14_19h48_08

When you click on the "Profil" button each element of the list is displayed one by one. Maybe the delay between each <LI> was so short then I updated it to 200ms.

**Comment by michael-albrecht-1** [comment]
Too many things changed I think it is better to do it in a new branch.  

ps : i splited navbar in two components (Navbar + Header) in the new branch install-styled-components

**Pull request closed without merging**